src/nni_manager/training_service/pai/paiData.ts

```diff
@@ -63,7 +63,7 @@ export const PAI_TRIAL_COMMAND_FORMAT: string =
 `export NNI_PLATFORM=pai NNI_SYS_DIR={0} NNI_OUTPUT_DIR={1} NNI_TRIAL_JOB_ID={2} NNI_EXP_ID={3} NNI_TRIAL_SEQ_ID={4}
 && cd $NNI_SYS_DIR && sh install_nni.sh 
 && python3 -m nni_trial_tool.trial_keeper --trial_command '{5}' --nnimanager_ip '{6}' --nnimanager_port '{7}' 
---pai_hdfs_output_dir '{8}' --pai_hdfs_host '{9}' --pai_user_name {10} --nni_hdfs_exp_dir '{11}'`;
+--pai_hdfs_output_dir '{8}' --pai_hdfs_host '{9}' --pai_user_name {10} --nni_hdfs_exp_dir '{11}' --nni_webhdfs_path '{12}'`;
```

Comment:
since hdfs options are general, could we remove all the specific prefixes? 
--pai_hdfs_output_dir  ->  --hdfs_output_dir
--pai_hdfs_host   ->  --hdfs_host
--nni_hdfs_exp_dir -> --hdfs_exp_dir
--nni_webhdfs_path -> --webhdfs_path

Reply:
updated to hdfs_output_dir, hdfs_host, hdfs_exp_dir, webhdfs_path